src/Traversals/Traversals.jl

```diff
@@ -350,4 +321,5 @@ include("randomwalks.jl")
 # export diffusion, diffusion_rate
 # export FixedColoring, RandomColoring, DegreeColoring, greedy_color
 # export mincut, maximum_adjacency_visit
+export  biconnected_components
```

Comment:
this should not be in Traversals, and in any case we're not ready to export anything from there right now.

Reply:
it is just a temporary situation, i just needed to make sure the algorithm works